### DISCUSSION TITLE
The mouse over the Tilemap's grid does not display its cell's coordinates?(Godot 4)

### DISCUSSION THREAD

**Discussion by KoromoHarisu** [godotengine/godot-proposals#6701] [2D]
In the previous version always can show a cell's coordinate in editor canvas's left bottom when my mouse over grids ,But i cant find any information for this in Godot4.0?

This is example of Godot3.0 cells coordinate display :

![Godot3_issue](https://user-images.githubusercontent.com/130833615/232208696-8d2e7149-0498-45ac-bda5-867485b4f505.png)

**Comment by Zireael07** [comment]
IIRC this is already tracked somewhere in the Tilemap improvements proposal?

**Comment by AThousandShips** [comment]
Tracked in #5792 and implemented in godotengine/godot#68800
